applications/webhooks/src/webhooks_channel_util.erl

```diff
@@ -11,7 +11,31 @@
 -include("webhooks.hrl").
 
 -spec handle_event(kz_json:object(), kz_proplist()) -> 'ok'.
-handle_event(JObj, _Props) ->
+handle_event(JObj, Props) ->
+    case kz_api:is_federated_event(JObj) of
+        'false' ->
+            continue_handle_event(JObj, Props);
+        'true' ->
+            EventZone = kz_api:event_zone(JObj),
+            WebHooksZones = props:get_value(<<"webhooks_zones">>, Props),
+            ZoneMatched = lists:member(EventZone, WebHooksZones),
+            ShouldHandle = props:get_is_true(<<"should_handle_federated">>, Props),
+            maybe_handle_federated_event(JObj, Props, ZoneMatched, ShouldHandle)
```

Comment:
in this group you only need to check `should_handle_federated`, the logic was already done in `handle_event`